web/src/lib/roadmap/buildUserRoadmap.test.ts

```diff
@@ -885,4 +887,40 @@ describe("buildUserRoadmap", () => {
       expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toEqual([]);
     });
   });
+
+  describe("raffle-bingo games", () => {
+    it("adds raffle-bingo-games task if non-profit business conducts raffle or bingo games", () => {
+      const profileData = generateStartingProfile({
+        raffleBingoGames: true,
+        legalStructureId: "nonprofit",
+      });
+
+      buildUserRoadmap(profileData);
+
+      expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).toContain("raffle-bingo-games");
+    });
+
+    it("does not add raffle-bingo-games task if non-profit business does not conduct raffle or bingo games", () => {
+      const profileData = generateStartingProfile({
+        raffleBingoGames: false,
+        legalStructureId: "nonprofit",
+      });
+
+      buildUserRoadmap(profileData);
+
+      expect(getLastCalledWith(mockRoadmapBuilder)[0].addOns).not.toContain("raffle-bingo-games");
+    });
+
+    it("does not add raffle-bingo-games task if business is not a non-profit", () => {
+      const filterNonprofitOut = LegalStructures.filter((x) => x.id !== "nonprofit");
+      const randomLegalStructure = randomElementFromArray(filterNonprofitOut);
+      buildUserRoadmap(
+        generateStartingProfile({
+          raffleBingoGames: false,
```

Comment:
[pebble] We know it won't get added if `raffleBingoGames` is `false`. Isn't the whole point that in the case this is true you'd still have to be a nonprofit to see the add-on?

```suggestion
          raffleBingoGames: true,
```